File: SesuatuApp/SesuatuApp/ViewController.m
//
//  ViewController.m
//  SesuatuApp
//
//  Created by Muhammad Fauzi Masykur on 12/05/22.
//

#import "ViewController.h"

@interface ViewController ()

@end

@implementation ViewController

- (void)viewDidLoad {
    [super viewDidLoad];
    // Do any additional setup after loading the view.
}

- (IBAction)startSDKPressed:(id)sender {
    
    SesuatuSDK *sesuatuSDK = [[SesuatuSDK alloc]init];
    [sesuatuSDK fetchDataFromApiWithCompletion:^(BoringResponse * _Nullable response , NSError * _Nullable error) {
        if (response) {
            NSLog(@"responsenya adalah %@", response.activity);
        } else {
            NSLog(@"errornya %@", error);
        }
    }];
}

@end

File: SesuatuAppSwift/SesuatuAppSwift/ViewController.swift
//
//  ViewController.swift
//  SesuatuAppSwift
//
//  Created by Muhammad Fauzi Masykur on 12/05/22.
//

import UIKit
import SesuatuSDK

class ViewController: UIViewController {

    override func viewDidLoad() {
        super.viewDidLoad()
        // Do any additional setup after loading the view.
    }


    @IBAction func startSDKPressed(_ sender: Any) {
        var sesuatuSDK = SesuatuSDK()
        sesuatuSDK.fetchDataFromApi { result, error in
            if (result != nil) {
                print("datanya \(result?.activity)")
            } else {
                print("error")
            }
        }
    }
}

